### ISSUE TITLE
TlpRemoteIDE rename

### ISSUE THREAD

**Issue by philippeback** [dionisiydk/TelePharo#6]
Now code that reads:

remotePharo := TlpRemoteIDE connectTo: (....)

is annoying with the "Remote" because it somewhat implies that the IDE is remote.
But the IDE is local and connects to a remote.

Maybe

remotePharo := TlpIDE connectToRemote: (...)

is more intention revealing.

**Comment by dionisiydk** [comment]
Good point. But I made many videos where I show this name. So current name should exists for some period. 
Auto deprecation of classes can help here. I hope we will have it in Pharo 7
